media-libs/libsdl3/libsdl3-3.2.4.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Simple Direct Media Layer 3.0"
+DESCRIPTION="Simple DirectMedia Layer is a cross-platform development library \
+designed to provide low level access to audio, keyboard, mouse, joystick, and \
+graphics hardware via OpenGL and Direct3D. It is used by video playback \
+software, emulators, and popular games."
+HOMEPAGE="https://www.libsdl.org/"
+COPYRIGHT="1997-2025 Sam Lantinga"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://www.libsdl.org/release/SDL3-$portVersion.tar.gz"
+CHECKSUM_SHA256="2938328317301dfbe30176d79c251733aa5e7ec5c436c800b99ed4da7adcb0f0"
+SOURCE_DIR="SDL3-$portVersion"
+# PATCHES=""
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+    libsdl3$secondaryArchSuffix = $libVersionCompat
+    lib:libsdl3$secondaryArchSuffix = $libVersionCompat
+    "
+REQUIRES="
+    haiku$secondaryArchSuffix
+    lib:libGL$secondaryArchSuffix
+    "
+
+PROVIDES_devel="
+    libSDL3${secondaryArchSuffix}_devel = $portVersion
+    devel:libSDL3$secondaryArchSuffix = $libVersionCompat
+    devel:libSDL3_test$secondaryArchSuffix = $portVersion
+    "
+REQUIRES_devel="
+    libSDL3$secondaryArchSuffix == $portVersion base
+    "
+
+BUILD_REQUIRES="
+    haiku${secondaryArchSuffix}_devel
+    devel:libGL$secondaryArchSuffix
+    "
+BUILD_PREREQUIRES="
+    cmd:cmake
+    cmd:gcc$secondaryArchSuffix
+    cmd:ld$secondaryArchSuffix
+    cmd:ninja
+    cmd:pkg_config$secondaryArchSuffix
+    "
+
+BUILD()
+{
+    cmake . -Bbuild -GNinja $cmakeDirArgs \
+        -DSDL_STATIC=OFF \
+        -DCMAKE_BUILD_TYPE=Release
+    ninja -C build
+}
+
+INSTALL()
+{
+    ninja -C build install
+
+    sed -i 's,${_IMPORT_PREFIX}/lib',$developDir, \
```

Comment:
I think this should be 
```suggestion
    sed -i 's,${_IMPORT_PREFIX}/lib',$developLibDir, \
```
but to be sure that should be tested with some other package which actually uses this cmake file.

Reply:
Understood. I will try with SDL3_image

Reply:
$developLibDir seems to work